<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>교란순열</string>
  </property>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10번</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_100">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>100번</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1000">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1000번</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_trial1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>전체 시행 횟수</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_acnt1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사건의 횟수</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_prob1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>확률</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_exN1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>이론값(1/e)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_error1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>오차(%)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>105</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>105</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>105</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>105</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>105</y>
     <width>70</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_trial">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_acnt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_prob">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_exN">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_des">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>93</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>설명 보기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
